.changeset/silly-buckets-joke.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-msgraph': patch
+---
+
+Fixed bug in readProviderConfig as it wasn't reading the user.select field expected from the app-config.yaml configuration
```

Comment:
I have a feeling that Vale the "spell checker" might not like some of these. The easy fix is to just use backticks (`) around: readProviderConfig, user.select and app-config.yaml. There's a GitHub action for this that will let you know for sure which words

Reply:
Looks like I'm happily wrong :)